contracts/VaultV3.vy

```diff
@@ -108,8 +108,8 @@ event UpdateProfitMaxUnlockTime:
 event Shutdown:
     pass
 
-event Sweep:
-    token: indexed(address)
+event DebtBought:
```

Comment:
q:
does "DebtPurchased" sound a little better + in line with "Purchaser" role?

Reply:
changed